include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort_one_wg.h

```diff
@@ -74,6 +74,9 @@ struct __subgroup_radix_sort
         uint16_t __buf_size;
 
       public:
+
+        static constexpr sycl::access::fence_space _space = sycl::access::fence_space::local_space;
```

Comment:
I think the problem with this approach is that `sycl::access::fence_space` is not a part of SYCL2020 at all, so we cannot expect a compiler to implement it.

Reply:
I think it's simple not documented now, but not deprecated too.
And we haven't any warnings about that.

Reply:
Practically we are using this enum in oneDPL code in main branch too, but only in one place.

Reply:
I see, so I guess the current state is probably not compatible with non-dpcpp SYCL compilers unless they also have this SYCL 1.2.1 support.

Reply:
May be. But in any case, we are not making anything worse at this place.